src/com/massivecraft/massivetickets/cmd/CmdTicketsCreate.java

```diff
@@ -61,6 +62,12 @@ public void perform() throws MassiveException
 		boolean update = msender.hasMessage();
 		String verb = update ? "updated" : "created";
 		
+		// But only if the ticket has been changed
+		if (update && MUtil.equals(msender.getMessage(), message))
+		{
+			msg("<b>Please do not spam our ticket system. Someone will be with you shortly.");
+			return;

```

Comment:
This is a question of preference, but you could also throw a MassiveException with the message.

Reply:
I had originally thrown a massive exception, but then I realized that we don't seem to do that very often in the commands here, so I went with the message to conform.